### PR TITLE
Adjust all tabs to utilize same style

### PR DESCRIPTION
**Pull request by NakedFool**
## What does this PR do and why?

Fixes - https://github.com/jbx-protocol/juice-interface/issues/2194

We already have shared `Tab` component that has been used in projects and in safe, now we are utilizing it in the funding cycle section.

I've adjusted small corrections to the code so `label` can be `string | JSX.Element` so we accept those warning icons and it leaves room for labels to be more creative if needed.

## Screenshots or screen recordings

https://user-images.githubusercontent.com/18723426/198573241-b953ce29-34d4-42dc-997e-89a2a8ac6818.mov

## Acceptance checklist

- [x] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [x] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [x] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

File: src/components/Project/FundingCycleSection.tsx [hunk 1]
```diff
@@ -55,23 +51,14 @@ export default function FundingCycleSection({
         {reconfigureButton}
       </div>
 
-      <Space style={{ fontSize: '.8rem', marginBottom: 12 }} size="middle">
+      <Space style={{ fontSize: '.8rem', marginBottom: 20 }} size="middle">
```
Comment (johnnyd-eth) [inline src/components/Project/FundingCycleSection.tsx:54]: **One small nit-pick:** I think we should make this `size="large"` to match the other tab instances.
(Safe has more space):
Screen Shot 2022-10-31 at 10 55 35 am
Screen Shot 2022-10-31 at 10 55 31 am

**Comment by johnnyd-eth** [comment]
Great pickup @NakedFool. Just one tiny comment!

File: src/components/Project/FundingCycleSection.tsx (hunk 1 above)
Comment (NakedFool) [reply src/components/Project/FundingCycleSection.tsx]: yes, agreed! looks much better, pushing!

Comment (johnnyd-eth) [reply src/components/Project/FundingCycleSection.tsx]: Sweet!

**Pull request merged**